runtime/tutor/tutor.tr.iso9

```diff
@@ -22,7 +22,7 @@
   �imdi Caps Lock d��menizin bas�l� olmad���na emin olun ve Ders 1.1'in
   ekran� tamamen doldurmas� i�in j d��mesine yeterli miktarda bas�n.
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
English version of tutor doesn't have a space here. Do you intentionally insert this space?

Reply:
Yes. One space at each side, thus providing an equal look, and better visual understanding of a new chapter.